api/src/datasets.py

```diff
@@ -51,6 +51,8 @@
 
 es = Elasticsearch([settings.ELASTICSEARCH_URL], port=settings.ELASTICSEARCH_PORT)
 
+tds_api = settings.TDS_API
```

Comment:
This variable name is a bit too ambiguous for my taste. I'd prefer something along the lines of `tds_host`. But really, I think that what we want is a URL because once deployed, things can be https and/or running on a different port, so it should be everything in the URL up to, but not including, the first slash in the url.

Reply:
That makes sense to me, I wasn't particularly excited about the name either. I will make this change.